src/_P020_Ser2Net.ino

```diff
@@ -322,11 +322,18 @@ boolean Plugin_020(uint8_t function, struct EventStruct *event, String& string)
         break;
       }
 
-      if (equals(command, F("serialsend"))) {
-        task->ser2netSerial->write(string.substring(11).c_str());
-        task->ser2netSerial->flush();
-        success = true;
-      }
+        if (equals(command, F("serialsend"))) {
+          task->ser2netSerial->write(string.substring(11).c_str());
+          task->ser2netSerial->flush();
+          success = true;
+        } else
+
+        if (equals(command, F("serialsendmix"))) {
+          String argument = parseHexTextString(string);
+          task->ser2netSerial->write(argument.c_str());
```

Comment:
@tonhuisman does it handle also 0x00 bytes ? shouldn't be like `->write(buf,size);`

Reply:
Handling `0x00` isn't planned, as that was never supported.
Unless this is your request to add that as a feature?

Reply:
@tonhuisman @TD-er  If you can input hex and you can put 0x00 inside and you don't plan to support it this will generate future issues filled on github ;)

Reply:
Nah, I think it will file `zero` issues ;)

Reply:
I'll check if sending out `\0` via serial will work as intended, and when positive see if I can implement it.

Reply:
Problem isn't that you can't send it via Serial.
But that all kinds of `String` functions look for a `\0` while others look for the `.size()` call.

Reply:
@TD-er exactly!

Reply:
I have added `parseHexTextData()` that returns a `std::vector<uint8_t>`, so `0x00` is now supported.